deps/oob-1.0.2/jsonutil.c

```diff
@@ -60,8 +66,10 @@ char *prepareOOBevent(oobOcspData *ocspevent)
   cJSON *root = NULL;
   cJSON *list = NULL;
   cJSON *tags = NULL;
-  cJSON *vals = NULL; 
-  cJSON *key  = NULL;
+  cJSON *vals = NULL;
+  cJSON *dns = NULL;
```

Comment:
dns or dsn ?

Reply:
DNS

Reply:
I'm confused by the naming. DNS stands for domain name system right? What DSN stands for? It seems the info includes all connection settings from ODBC right? Why not call it `connectionSettings`?

Reply:
It should be DSN which mean Data Source Name

Reply:
aha, DSN should be database source name, that makes sense!

Reply:
Ok, I can change this back.

Reply:
Yes